docs/integrations/openvino.md

```diff
@@ -0,0 +1,229 @@
+| yolov8m | OpenVINO    | ✅      | 49.8      | 0.4997      | 7.11                   |
+| yolov8l | PyTorch     | ✅      | 83.7      | 0.5293      | 77.45                  |
+| yolov8l | TorchScript | ✅      | 167.2     | 0.5268      | 85.71                  |
+| yolov8l | ONNX        | ✅      | 166.8     | 0.5268      | 88.94                  |
+| yolov8l | OpenVINO    | ✅      | 167.0     | 0.5264      | 9.37                   |
+| yolov8x | PyTorch     | ✅      | 130.5     | 0.5404      | 100.09                 |
+| yolov8x | TorchScript | ✅      | 260.7     | 0.5371      | 114.64                 |
+| yolov8x | ONNX        | ✅      | 260.4     | 0.5371      | 110.32                 |
+| yolov8x | OpenVINO    | ✅      | 260.6     | 0.5367      | 15.02                  |
+
+#### Flex GPU (FP16 Precision)
+
+Benchmark results for Flex GPU at FP16 precision with `half=True`:
+
+<div align="center">
+<img width="640" src="https://user-images.githubusercontent.com/26833433/252764828-442bd1c9-e52c-426f-b31c-98e88a887d9c.jpg">
+</div>
+
+| Model   | Format      | Status | Size (MB) | mAP50-95(B) | Inference time (ms/im) |
+|---------|-------------|--------|-----------|-------------|------------------------|
+| yolov8n | PyTorch     | ✅      | 6.2       | 0.3709      | 24.39                  |
+| yolov8n | TorchScript | ✅      | 12.4      | 0.3704      | 23.76                  |
+| yolov8n | ONNX        | ✅      | 12.2      | 0.3704      | 37.37                  |
+| yolov8n | OpenVINO    | ✅      | 6.3       | 0.3703      | 3.29                   |
+| yolov8s | PyTorch     | ✅      | 21.5      | 0.4471      | 30.60                  |
+| yolov8s | TorchScript | ✅      | 42.9      | 0.4472      | 34.40                  |
+| yolov8s | ONNX        | ✅      | 42.8      | 0.4472      | 43.53                  |
+| yolov8s | OpenVINO    | ✅      | 21.6      | 0.4470      | 3.91                   |
+| yolov8m | PyTorch     | ✅      | 49.7      | 0.5013      | 50.75                  |
+| yolov8m | TorchScript | ✅      | 99.2      | 0.4999      | 47.90                  |
+| yolov8m | ONNX        | ✅      | 99.0      | 0.4999      | 63.16                  |
+| yolov8m | OpenVINO    | ✅      | 49.8      | 0.4997      | 7.11                   |
+| yolov8l | PyTorch     | ✅      | 83.7      | 0.5293      | 78.55                  |
+| yolov8l | TorchScript | ✅      | 167.2     | 0.5268      | 84.75                  |
+| yolov8l | ONNX        | ✅      | 166.8     | 0.5268      | 87.71                  |
+| yolov8l | OpenVINO    | ✅      | 83.8      | 0.5264      | 9.38                   |
+| yolov8x | PyTorch     | ✅      | 130.5     | 0.5404      | 94.85                  |
+| yolov8x | TorchScript | ✅      | 260.7     | 0.5371      | 113.11                 |
+| yolov8x | ONNX        | ✅      | 260.4     | 0.5371      | 111.00                 |
+| yolov8x | OpenVINO    | ✅      | 130.6     | 0.5367      | 15.02                  |
```

Comment:
As the results are almost the same, I'd go here with FP32 only.

Reply:
Ok got it. I'll remove Flex FP16.

Reply:
Done in https://github.com/ultralytics/ultralytics/pull/3624/commits/c694d451a357218ddb8bf2794c6625b34c08f919